packages/core-datepicker/core-datepicker.js

```diff
@@ -57,10 +57,22 @@ addEvent(UUID, 'keydown', (event) => {
 })
 
 function onChange ({target}) {
```

Comment:
denne funksjonen er kanskje strålende, men her må jeg tenke veldig veldig lenge for å skjønne hva som skjer og hvorfor det gjøres.

ønsker meg generelt sett mer verbos kode med tilhørende tester.

tror kanskje ikke jeg er rett person for review av `core-datepicker.js`. 🤗

Reply:
Hehe, enig - nåværende kode er er to the point og iterert av behov og uu quirks som dukker opp - det er på todo å gå gjennom og bryte opp litt mer, mer kommentarer og funksjoner, noe mer ut til utils og skrive tester 👍